lib/esp/resources/metadata.rb

```diff
@@ -32,12 +36,14 @@ def self.for_alert(alert_id = nil)
 
     # Find a Metadata object by id
     #
-    # ==== Parameter
-    #
-    # +id+ | Required | The ID of the metadata object to retrieve
+    # *call-seq* -> +super.find(id)+
     #
-    # :call-seq:
-    #  find(id)
+    # @overload find(id)
+    #   @param id [Integer, Numeric, #to_i] Required ID of the metadata object to retrieve.
+    # @overload find(non_numeric, options = {})
```

Comment:
Not sure about documenting this overload, but if we do, it whould be `@overload find(options = {})`